circle.yml

```diff
@@ -2,6 +2,9 @@ machine:
   python:
     version: 2.7.6
 
+dependencies:
+  override:
+    - pip install tox
```

Comment:
tox is already declared in setup.py as a thing required by the tests. If you run the tests as `python setup.py test` (which CircleCI does by default I bet, if you don't override it below), then your test dependencies will get installed automatically.